packages/fxa-auth-server/package.json

```diff
@@ -141,6 +148,7 @@
     "load-grunt-tasks": "^5.1.0",
     "lodash.chunk": "4.2.0",
     "lodash.pick": "4.4.0",
+    "maildev": "^1.1.0",
```

Comment:
If you install maildev globally (`yarn global add maildev`) do we need maildev as a project dependency?

Reply:
Yeah we don't need it. Thanks for pointing it out though I was trying to get a `maildev` server up with the help of a javascript file but since it's mentioned in the [docs](https://github.com/mozilla/fxa#running-with-maildev) to manually get it up, I stopped looking into it and probably forgot to remove that dependency.